### PR TITLE
Implement change_table_comment and change_column_comment

### PR DESCRIPTION
**Pull request by alecspopa**
Implement change_table_comment and change_column_comment for MySql.
The two methods were defined in PR https://github.com/rails/rails/pull/22911 but only implemented for PostgreSQL.

### CODE REVIEW

File: activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb [hunk 1]
```diff
@@ -365,6 +370,12 @@ def change_column_null(table_name, column_name, null, default = nil) #:nodoc:
         change_column table_name, column_name, column.sql_type, null: null
       end
 
+      # Adds comment for given table column or drops it if +comment+ is a +nil+
+      def change_column_comment(table_name, column_name, comment)
```
Comment (kamipo) [inline activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb]: Add `# :nodoc:`.

File: activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb [hunk 2]
```diff
@@ -311,6 +311,11 @@ def bulk_change_table(table_name, operations) #:nodoc:
         execute("ALTER TABLE #{quote_table_name(table_name)} #{sqls}")
       end
 
+      # Adds comment for given table or drops it if +comment+ is a +nil+
+      def change_table_comment(table_name, comment)
```
Comment (kamipo) [inline activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb]: Add `# :nodoc:`.

**Review by kamipo — COMMENTED** [review]
Can you add tests that comment is dropped if `comment` is `nil`?

**Comment by alecspopa** [comment]
Sure, I'll add the test for update to nil.

**Comment by kamipo** [comment]
Can you squash your commits into 1 please?

**Comment by alecspopa** [comment]
Done. Thank you for your review.

**Comment by kamipo** [comment]
Thanks!
